src/renderer/src/routes/pages/Wallets/index.tsx

```diff
@@ -182,6 +229,69 @@ export const WalletsPage = () => {
           </footer>
         </section>
       )}
-    </PortfolioLayout>
+      <div className="flex-grow flex flex-col gap-y-5">
+        <div className="w-full flex bg-gray-800 rounded flex-grow drop-shadow-lg animate-pulse"></div>
+        <div className="w-full bg-gray-800 rounded flex-grow drop-shadow-lg overflow-scroll max-h-[20rem]">
+          <h1 className="text-white m-4">{commonActivity('title')}</h1>
+          <Separator className="mx-4" />
+          <InfiniteScroll
+            dataLength={transactions.length}
+            next={() => {}}
+            hasMore={true}
+            loader={<h4>Loading...</h4>}
+            endMessage={
+              <p style={{ textAlign: 'center' }}>
+                <b>Yay! You have seen it all</b>
```

Comment:
Put this inside the lang

Reply:
This won't be the final one; adjustments are needed after the react-query above.